docs/reference/rest-api/security/bulk-update-api-keys.asciidoc

```diff
@@ -63,6 +63,9 @@ The structure of a role descriptor is the same as the request for the <<api-key-
 Within the `metadata` object, top-level keys beginning with an underscore (`_`) are reserved for system usage.
 Any information specified with this parameter fully replaces metadata previously associated with the API key.
 
+`expiration`::
+(Optional, string) Expiration time for the API keys. By default, API keys never expire.
```

Comment:
Higher up in the docs, there's a sentence "This API supports updates to API key access scope and metadata" -- lets tweak that to reflect the change. Same for the other API docs.

Could also add that you can omit `expiration` to leave one previously assigned unchanged.